frontend/src/components/AccountSendTable.vue

```diff
@@ -0,0 +1,184 @@
+          <div v-if="col.name === 'dateSent'" class="d-inline-block">
+            <div
+              @click="openInEtherscan(props.row.hash)"
+              class="row justify-start items-center cursor-pointer external-link-icon-parent"
+            >
+              <div class="col-auto">
+                <div>{{ col.value }}</div>
+                <div class="text-caption text-grey">{{ props.row.dateSentTime }}</div>
+              </div>
+              <q-icon class="external-link-icon" name="fas fa-external-link-alt" right />
+            </div>
+          </div>
+
+          <!-- Amount column -->
+          <div v-else-if="col.name === 'amount'">
+            <div class="row justify-start items-center no-wrap">
+              <img class="col-auto q-mr-md" :src="props.row.tokenLogo" style="width: 1.2rem" />
+              <div class="col-auto">
+                {{ col.value }}
+                {{ props.row.tokenSymbol }}
+              </div>
+            </div>
+          </div>
+
+          <!-- Sender column -->
+          <div v-else-if="col.name === 'from'" class="d-inline-block">
+            <div @click="copyAddress(props.row.address)" class="cursor-pointer copy-icon-parent">
+              <span>{{ props.row.addressShortened }}</span>
+              <q-icon class="copy-icon" name="far fa-copy" right />
+            </div>
+          </div>
+          <!-- Default -->
+          <div v-else>{{ col.value }}</div>
+        </q-td>
+      </q-tr>
+    </template>
+  </q-table>
+</template>
+
+<script lang="ts">
+import { defineComponent, getCurrentInstance, PropType } from '@vue/composition-api';
+import BaseTooltip from 'src/components/BaseTooltip.vue';
+import { copyToClipboard } from 'quasar';
+import { notifyUser } from 'src/utils/alerts';
+import { toAddress } from 'src/utils/address';
+import { getEtherscanUrl } from 'src/utils/utils';
+import useWalletStore from 'src/store/wallet';
+
+type SendTableMetdataRow = {
+  dateSent: string;
+  dateSentUnix: number;
+  dateSentTime: string;
+  amount: string;
+  address: string;
+  hash: string;
+  tokenLogo?: string;
+  tokenAddress: string;
+  tokenSymbol: string;
+};
+
+export default defineComponent({
+  name: 'AccountSendTable',
+  components: { BaseTooltip },
+  props: {
+    sendMetadata: {
+      type: (undefined as unknown) as PropType<SendTableMetdataRow[]>,
+      required: true,
+    },
+  },
+
+  setup(props, context) {
+    const vm = getCurrentInstance()!;
+    const { provider, chainId } = useWalletStore();
+    const paginationConfig = { rowsPerPage: 25 };
+    async function copyAddress(address: string) {
+      if (!provider.value) throw new Error(vm.$i18n.tc('AccountSendTable.wallet-not-connected'));
+      const mainAddress = await toAddress(address, provider.value);
+      await copyToClipboard(mainAddress);
+      notifyUser('success', `${vm.$i18n.tc('AccountSendTable.address-copied')}`);
+    }
+    function openInEtherscan(hash: string) {
```

Comment:
This one is similarly defined twice, but maybe ok since `getEtherscanUrl` is the heavy lifting